### PR TITLE
Cleanup README

### PR DESCRIPTION
**Pull request by stv0g**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,20 +218,19 @@ There are 2 common problems:
 * The required codec library is not installed (vpx in this example). In this case, please refer to the [available codecs](#available-codecs).
 * Pkg-config fails to find the `.pc` files for this codec ([reference](https://people.freedesktop.org/~dbn/pkg-config-guide.html#using)). In this case, you need to find where the codec library's `.pc` is stored, and let pkg-config knows with: `export PKG_CONFIG_PATH=/path/to/directory`.
 
+### Roadmap
+The library is used as a part of our WebRTC implementation. Please refer to that [roadmap](https://github.com/pion/webrtc/issues/9) to track our major milestones.
```
Comment (at-wat) [inline README.md:222]: I believe MediaDevices API is not a *part* of WebRTC but can be used with. 
- Web APIs
  - WebRTC https://developer.mozilla.org/en-US/docs/Web/API/WebRTC_API
  - Media Capture and Streams API https://developer.mozilla.org/en-US/docs/Web/API/Media_Capture_and_Streams_API
  - ...

Comment (stv0g) [reply README.md]: Agreed :) I changed the wording :)

File: README.md [hunk 2]
```diff
@@ -19,7 +19,7 @@
 
 `go get -u github.com/pion/mediadevices`
 
-## Usage
+### Usage
```
Comment (at-wat) [inline README.md:22]: Hierarchy of the sections are changed to
```
## Install
### Usage
### More Examples
### Available Media Inputs
### Available Codecs
#### Video Codecs
##### x264
### FAQ
#### Failed to find the best driver that fits the constraints
### Roadmap
### Community
### Contributing
### License
```
but I don't think these '###' sections are subsections of `## Install`

Comment (stv0g) [reply README.md:24]: Oh, I missed the install section. Thanks. I fixed it now.

All Pion repos are using level-3 sections in the readme as the highest level (`###`).

**Review by at-wat — APPROVED** [review]

**Pull request merged**